larray/viewer/api.py

```diff
@@ -155,7 +155,26 @@ def view(obj=None, title='', depth=0):
 
 
 def compare(*args, **kwargs):
+    """
```

Comment:
If users ask, we will think about it. In the meantime, let's go with this PR as it is.